save-cloud-common/src/jvmMain/kotlin/org/cqfn/save/entities/Agent.kt

```diff
@@ -14,7 +14,7 @@ import javax.persistence.ManyToOne
 class Agent(
     var containerId: String,
 
-    @ManyToOne(fetch = FetchType.LAZY)
+    @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
Why did you change it? Absence of this field should be fixed by applying `@Transactional` when necessary